misc/build-wyrm-transpiler.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -ex
+source common.sh
+
+VERSION=$1
+if [[ "${VERSION}" = "trunk" ]]; then
+    VERSION=trunk-$(date +%Y%m%d)
+    BRANCH=main
+else
+    BRANCH=V${VERSION}
```

Comment:
Doesn't look like yuo've had any releases yet (which is fine!) -- most github stuff uses a lower-case `v` prefix just as an observation.

Reply:
Thanks for the observation! I copied this from the movfuscator build script which I copied from something else I think, the branch isn't entered on either currently :)